### PR TITLE
slightly clearer language about breaks.

### PR DESCRIPTION
**Pull request by katyhuff**
## Summary of changes
It's recently become clear that the expectations doc doesn't quite capture my expectations about communicating vacations. While I'm never going to be interested in tracking student hours, I have always expected that students communicate times when they will be out of communication or when they plan to take a break from working on days that are not staff holidays. One expectation remains implicit rather than explicit: if we have a meeting scheduled on the group calendar, of course, it's simply common courtesy to let me know if you plan to miss it. 

## Types of changes
This change updates the expectations documentation on the website to make the vacation communication policy more clear. Specifically:

- [x] New feature (non-breaking change which adds functionality)
- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.

## Associated Issues and PRs
n/a

## Associated Developers

- Dev: @achubbz 
- Dev: @gtw2 
- Dev: @gwenchee 
- Dev: @andrewryh 
- Dev: @jbae11 
- Dev: @robfairh 
- Dev: @smpark7 
- Dev: @kamuda1 
- Dev: @ZoeRichter

### CODE REVIEW

File: manual/expectations.md [hunk 1]
```diff
@@ -61,9 +61,14 @@ As your PI, it is my responsibility to do a number of things.
...
   Feel free to follow a schedule that works for you, work from home once in a 
-  while when you need to, and take breaks when you want to. But, by default, 
-  plan to come to the office during the daytime with regularity.  I won't track 
-  your hours, but I will worry if I haven't seen your face all week.
+  while when you need to, and take breaks when you need to. But, by default, 
+  plan to come to the office during the daytime with regularity.  
+  I won't track your hours, but I do want to know if you'll be 
+  out-of-communication, and I will worry if I haven't seen your face all week.
+  If you plan to take a vacation, please let me know in advance, particularly 
+  if we have meetings scheduled while you'll be away. Postdocs are considered
```
Comment (jbae11) [inline manual/expectations.md]: Graduate students are considered staff as well, right?

Comment (katyhuff) [reply manual/expectations.md]: yes, as below.

Comment (katyhuff) [reply manual/expectations.md]: (this section is about postdocs... the section below is about grad students... it also includes the text about being staff...)

**Comment by andrewryh** [comment]
Looks clear for me

**Comment by jbae11** [comment]
My bad. Yeah looks clear to me too

**Pull request merged**